tests/appveyor/mariadb.appveyor.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It's a placeholder, I'm looking into the possibility of installing MariaDB in appveyor as there is a windows version.